server/src/org/apache/cloudstack/affinity/AffinityGroupServiceImpl.java

```diff
@@ -323,12 +330,12 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
         List<String> types = new ArrayList<String>();
 
         for (AffinityGroupProcessor processor : _affinityProcessors) {
-                if (processor.isAdminControlledGroup()) {
-                    continue; // we dont list the type if this group can be
-                              // created only as an admin/system operation.
-                }
-                types.add(processor.getType());
+            if (processor.isAdminControlledGroup()) {
```

Comment:
Are these formatting changes needed?
Are you using the CloudStack code formatter style? Or was the code not formatted properly?